pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -123,6 +123,13 @@ type SourceResult struct {
 type BundleSourceResult struct {
 	// Digest hold the image digest result
 	Digest string `json:"digest,omitempty"`
+
+	// ImageTimestamp holds the (bundle) image timestamp result
+	ImageTimestamp string `json:"image_timestamp,omitempty"`
+
+	// SourceTimestamp holds the source timestamp, meaning the most recent
+	// timestamp of all files that are used as source
+	SourceTimestamp string `json:"source_timestamp,omitempty"`
```

Comment:
I am in favor of a generic timestamp as property inside SourceResult.

Also, I'd like the fields to be of type `*metav1.Time` like `.status.startTime`.

Reply:
Ok, let me look into that.